src/i18n.js

```diff
@@ -25,7 +25,8 @@ export async function loadLanguage() {
         escapeValue: false // react already safes from xss
       },
       keySeparator: '>',
-      nsSeparator: '|'
+      nsSeparator: '|',
+      compatibilityJSON: 'v3',
```

Comment:
Plurals werent working and I found this issue here: https://github.com/i18next/i18next-scanner/issues/228
As I couldnt make the parser work, I thought it would be fine to switch to back to v3 as we probably will not need complicated plural forms..

Reply:
hmm, weblate also uses v4 for the translations (which probably is fine most of the times). However, as per the issue i18next-scanner seems to be unmaintained / not keeping up with i18next we should probably switch to i18next-parser ? :/